src/main/resources/config/en.yml

```diff
@@ -85,6 +85,8 @@ Experiment_WebhookChatMessageAvatarFromDiscord: false
 # Embed & webhook image/avatar url format
 # Alternative embed avatar url for offline servers:
 # https://cravatar.eu/helmavatar/{username}/{size}.png#{texture}
+# Use this embed avatar url if you are using GeyserMC:
+# https://api.tydiumcraft.net/v1/players/skin?type=avatar&uuid={uuid-nodashes}
```

Comment:
We need the message to be in all languages. I'd suggest just shortening your comment to `GeyserMC: https://api.tydiumcraft.net/v1/players/skin?type=avatar&uuid={uuid-nodashes}`

```suggestion
# GeyserMC: https://api.tydiumcraft.net/v1/players/skin?type=avatar&uuid={uuid-nodashes}
```